Console.Calculator.oksanatk/Program.cs

```diff
@@ -0,0 +1,390 @@
+﻿using Spectre.Console;
+using CalculatorLibrary;
+using Microsoft.CognitiveServices.Speech;
```

Comment:
🟢 **Speech Services**

⭐ Fantastic implementation and makes this project stand out from the rest! Well done 🥇